services/src/main/scala/cromwell/pipeline/service/ProjectFileService.scala

```diff
@@ -32,11 +33,20 @@ class ProjectFileService(
 
   def buildConfiguration(
     projectId: ProjectId,
-    projectFile: ProjectFile
-  ): Future[Either[ValidationError, ProjectConfiguration]] =
-    Future(womTool.inputsToList(projectFile.content.content)).map {
-      case Left(value) => Left(ValidationError(value.toList))
-      case Right(nodes) =>
-        Right(ProjectConfiguration(projectId, active = true, List(ProjectFileConfiguration(projectFile.path, nodes))))
+    projectFilePath: Path,
+    version: Option[PipelineVersion],
+    userId: UserId
+  ): Future[Either[Exception, ProjectConfiguration]] =
```

Comment:
I just noticed that return type is `Future[Either[Exception, _]]`

I think you should use `Future[_]` instead. `Future[Either[_, _]]` makes sense when you want to explicitly show that you expect some specific errors. But `Future[Either[Exception, _]]` is almost as vague as `Future[Either[Throwable, _]]` which can be simplified to `Future[_]`.

Reply:
Fixed.